app/javascript/gobierto_data/webapp/components/sets/SQLEditorCode.vue

```diff
@@ -108,32 +112,44 @@ export default {
     this.code = `SELECT * FROM ${this.tableName}`
   },
   mounted() {
-    this.cm = this.$refs.myCm.codemirror;
-    this.commands = commands;
-    const tables = {
-      table_1: [],
-      table_2: [],
-      table_3: [],
-      table_4: [],
-      table_5: [],
-      table_6: []
-    }
-    /*window.addEventListener('keydown', e => {
-      if (e.keyCode == 32) {
-        this.formatCode();
-        this.cm.setCursor(this.cm.lineCount(), 0);
-        this.cm.setCursor(this.cm.lineCount(), 0);
+    this.mergeTables()
+
+    this.editor = CodeMirror.fromTextArea(this.$refs.myCm, this.cmOption
+    )
+
+    this.editor.on("keypress", editor => {
+      editor.showHint()
+      this.$root.$emit('activeSave', false);
+      if (this.saveQueryState === true) {
+        this.$root.$emit('updateActiveSave', true, false);
       }
     })
-    */
 
-    this.cmOption.hintOptions.tables = tables;
+    this.editor.on('focus', editor => {
```

Comment:
Have you tried to apped this event to the textarea with `@focus`, `@change`?

Reply:
Yes, and it doesn't work :\